simplified-app-shared/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,7 @@
   <string name="catalog_refresh">Refresh Feed</string>
   <string name="catalog_search">Search</string>
   <string name="catalog_unknown_feed">Unknown</string>
+  <string name="catalog_more_feed">More...</string>
```

Comment:
@aferditamuriqi: We should use the actual ellipsis character in such strings ("…"), not three periods ("...").